pkg/terminal/parser.go

```diff
@@ -136,10 +147,21 @@ func (m *Parser) parse() ([]byte, error) {
 				m.pos = begin - 1
 
 			case ELINE, SGR:
-				m.data = sliceEdit(m.data, escStart, m.pos+1)
+				// Erases part of the line.
+				// If n is 0 (or missing), clear from cursor to the end of the line.
```

Comment:
What is an 'n'?

Reply:
"n" stands for number value. Full specification is "CSI n K"